main.go

```diff
@@ -1,71 +1,9 @@
 package main
 
 import (
-	"log"
-
-	endure "github.com/spiral/endure/pkg/container"
-	"github.com/spiral/roadrunner/v2/plugins/gzip"
-	"github.com/spiral/roadrunner/v2/plugins/headers"
-	"github.com/spiral/roadrunner/v2/plugins/static"
-	"github.com/spiral/roadrunner/v2/plugins/status"
-
-	// plugins
 	"github.com/spiral/roadrunner-binary/v2/cli"
-	httpPlugin "github.com/spiral/roadrunner/v2/plugins/http"
-	"github.com/spiral/roadrunner/v2/plugins/informer"
-	"github.com/spiral/roadrunner/v2/plugins/logger"
-	"github.com/spiral/roadrunner/v2/plugins/metrics"
-	"github.com/spiral/roadrunner/v2/plugins/reload"
-	"github.com/spiral/roadrunner/v2/plugins/resetter"
-	"github.com/spiral/roadrunner/v2/plugins/rpc"
-	"github.com/spiral/roadrunner/v2/plugins/server"
-	"github.com/temporalio/roadrunner-temporal/activity"
-	temporalClient "github.com/temporalio/roadrunner-temporal/client"
-	"github.com/temporalio/roadrunner-temporal/workflow"
 )
 
 func main() {
-	var err error
```

Comment:
I'm not sure I like the idea to move everything to root.go. It makes it impossible to build custom versions of RR and hardcode the plugins within the internals. While are not encouraging to build custom RR versions I'm seeing no reason to make it harder for users than it should be.

Reply:
There is zero difference. Because `Container` instance was shared between `main.go` <-> `root.go` because of the nature of `cobra` model. But putting plugins to the `main.go` has some disadvantages:
1. Not able to pass the config. At the moment parts of the container initialization lays in the `main.go`, part in the `root.go`. Now, all the parts located in the `root.go`.
2. Not able to configure endure container, because container instance located in the `main.go` which according to the p1 has no access to the config.